doc/scapy/installation.rst

```diff
@@ -190,18 +187,7 @@ Here are the topics involved and some examples that you can use to try if your i
  
 * VOIP. ``voip_play()`` needs `SoX <http://sox.sourceforge.net/>`_.
  
-* IPsec Crypto Support. ``SecurityAssociation()`` needs `Pycrypto <https://github.com/dlitz/pycrypto>`_. Combined AEAD modes such as GCM and CCM require pycrypto2.7a1, which is only available from source (no pip or package).
-
-  .. code-block:: text
-  
-    # pycrypto 2.6 install
-    sudo pip install pycrypto
-
-    # pycrypto 2.7a1 install
-    curl -sL https://github.com/dlitz/pycrypto/archive/v2.7a1.tar.gz | tar xz
-    cd pycrypto-2.7a1
-    python setup.py build
-    sudo python setup.py install
+* IPsec Crypto Support. ``SecurityAssociation()`` needs `Pycrypto 2.7a1 <https://github.com/dlitz/pycrypto>`_. Combined AEAD modes such as GCM and CCM are not available yet because of cryptography restrictions.
```

Comment:
No, because as it said "Combined AEAD modes such as GCM and CCM are not available yet because of cryptography restrictions.". We left a pycrypto part, see: https://github.com/secdev/scapy/blob/3f483a013cab1b3e59e831286d0ccb5ae69872c2/scapy/layers/ipsec.py#L164